pages.pt_BR/linux/abroot.md

```diff
@@ -0,0 +1,33 @@
+# abroot
+
+> O utilitário ABRoot fornece total imutabiliodade e atomicidade ao transacionar entre 2 estados da partição raíz (A⟺B).
+> Isso também permite transações sob demanda via um shell transacional.
+> Mais informações: <https://github.com/Vanilla-OS/ABRoot>.
+
+- Saída do estado da partição raiz atual ou futuro:
+
+`sudo abroot get {{presente|futuro}}`
+
+- Insere o shell transacional na partição raíz futura e troca a raíz na próxima inicialização:
+
+`sudo abroot shell`
```

Comment:
```suggestion
- Insira um shell transacional na futura partição raiz e alterne o root na próxima inicialização:

`sudo abroot shell`
```

Optional fix (error source: original page): unexpected `the` term, `a` expected.
Note: `Enter a transactional shell in the future root partition and switch root on the next boot:` can be used as a translation source.